src/applications/personalization/dashboard/containers/DashboardAppWrapper.jsx

```diff
@@ -63,6 +67,30 @@ class DashboardAppWrapper extends React.Component {
     }
   };
 
+  renderBreadcrumbs = location => {
```

Comment:
You might be able to inline this jsx using a [Fragment](https://reactjs.org/blog/2017/11/28/react-v16.2.0-fragment-support.html). I think the `<Breadcrumb/>` requires its children to be an array so depends on how Fragment works.

Reply:
Thanks for the reminder that we can start using Fragments now! In this case, since we are conditionally pushing an item onto the array of elements, I don't think there's a net win to using a Fragment, aside from being able to skip adding a `key` on each array element. I also just checked and although passing a Fragment full of elements to the Breadcrumbs kinda works, it renders all the elements as a single crumb. So the Breadcrumbs component appears to need some refactoring to support getting a Fragment as its child. _Finally_ we'll need to update to Babel 7 to get that sweet, sweet `<></>` syntax, which I know was something that was mentioned when you talked about the React 16 upgrade.

Reply:
I think the main benefit to using it would be getting rid of the partial renderer. Partials are imperative; JSX is easier to read if it's declarative.

Reply:
I'm working on a Formation Breadcrumb refactor that's more tolerant of individual items being added. Right now it's asking for the `React.Children.count(this.props.children)` to loop through and clone the last link so we can apply an `aria-current` attribute and wrap each one in a LI.

I could add a logic check that if the length was 1, or the child.type was fragment, to exit immediately and put the power back in the developer's hands to render the markup.

I'll have a PR for the work tomorrow and will tag you both for discussion.